apps/dash-image-segmentation/app.py

```diff
@@ -32,7 +32,7 @@
 # the number of different classes for labels
 NUM_LABEL_CLASSES = 5
 DEFAULT_LABEL_CLASS = 0
-class_label_colormap = px.colors.qualitative.Light24
+class_label_colormap = ["#E69F00", "#56B4E9", "#009E73", "#F0E442", "#0072B2"]
```

Comment:
Since we only have 5 categories, maybe you could use the "black" color tone from the paper. It seems to be the most distinct from the others.